tests/unit/cli/lookup-command-test.js

```diff
@@ -136,9 +137,10 @@ describe('cli/lookup-command.js', function () {
       ui,
     });
 
-    expect(ui.output).to.match(
-      /WARNING: An ember-addon has attempted to override the core command "serve"\. The core command will be used.*/
+    let re = new RegExp(
+      `WARNING: An ember-addon \\(${project.addons[0].name}\\) has attempted to override the core command "serve"\\. The core command will be used.*`
```

Comment:
I think using the string FakeAddon would be better here. I would hate for the code to not be working, but the test pass because `project.addons[0].name` was "correctly" undefined.

Reply:
There are 2 tests like this. I switched both of them to include the hardcoded addon name rather than the expression to fetch the name.

Reply:
Hmm I'm not seeing the change. Perhaps it was a bad push?